desktop/libs/azure/src/azure/client.py

```diff
@@ -66,7 +66,7 @@ def _make_adls_client(identifier):
 def _make_abfs_client(identifier):
   client_conf = conf.ABFS_CLUSTERS[identifier]
   azure_client = CLIENT_CACHE["azure"][identifier]
-  return ABFS.from_config(client_conf, azure_client)
+  return ABFS.from_config(client_conf, ActiveDirectory.from_config(None, version='v2.0'))#temp fix
```

Comment:
Need to fix?

Reply:
Currently, the variable "azure_client" does not have a version since it refers to CLIENT_CACHE["azure"][identifier]. So, abfs has an authentication error when trying to upload. In addition, editing CLIENT_CACHE["azure"][identifier] might affect adls since it also uses the same variable with its version set to "". Also, I am not sure if manually putting in ActiveDirectory.from_config is okay, since it ignores CLIENT_CACHE["azure"][identifier].

Reply:
We could add another cache client in the cache for AD v.2